node/config/def.go

```diff
@@ -114,6 +115,28 @@ type Pubsub struct {
 	RemoteTracer string
 }
 
+type Blockstore struct {
+	EnableSplitstore bool
+	Splitstore       Splitstore
+}
+
+type Splitstore struct {
+	HotStoreType         string
+	TrackingStoreType    string
+	MarkSetType          string
+	EnableFullCompaction bool
+	EnableGC             bool // EXPERIMENTAL
+	Archival             bool
+}
+
+func (s *Splitstore) GetHotStoreType() string {
+	// default is badger
+	if s.HotStoreType == "" {
+		return "badger"
+	}
+	return s.HotStoreType
+}
```

Comment:
If you set values in the default func below, it will be also used in configs which don't set this explicitly (so this wouldn't be needed)

Reply:
ah good point! thanks, will do.

Reply:
done.